app/lib/services/connect.ts

```diff
@@ -270,6 +275,8 @@ function connect({ server, logoutOnError = false }: { server: string; logoutOnEr
 			})
 		);
 
+		logoutListener = sdk.current.onStreamData('stream-force-logout', () => store.dispatch(logout(true)));
```

Comment:
This is from a different PR. Can you remove it?

Reply:
kek